prime-angular-frontend/src/app/modules/provisioner-access/pages/certificate/certificate.component.html

```diff
@@ -21,19 +21,38 @@
     </app-page-subheader>
 
     <div class="mb-4">
-
       <app-enrollee-property title="Name"
                              [makeBold]="true">
         {{ fullName }}
       </app-enrollee-property>
-
-      <app-enrollee-property title="User Type"
+      <app-enrollee-property title="Access Type"
                              [makeBold]="true">
         {{ userType }}
       </app-enrollee-property>
-
     </div>
 
+    <ng-container *ngFor="let licence of certificate.licences">
+      <app-enrollee-property title="Licence"
+                             [makeBold]="true">
+        <app-enrollee-property title="College ID"
+                               [makeBold]="true">
+          {{ licence.practRefId }}
+        </app-enrollee-property>
```

Comment:
For colleges that don't have a Pract Ref ID, can a "-" be displayed instead ... currently

![image](https://user-images.githubusercontent.com/7586665/184727111-ba9c32f1-be21-4750-b74c-1f42406531f6.png)

Reply:
Good idea. Logic is added to show "-" when no college ID.